src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,57 @@
+import styled from 'styled-components';
+import React from 'react';
+import PropTypes from 'prop-types'
+
+const FooterBox = styled.div`
+    background: #424242;
+    height: 30px;    
+    margin-top: 1rem;
+    padding: 1rem;
+    position: fixed;
+    bottom: 0;
+    left: 0;
+    width: 100%;    
+`;
+
+const StyledH1 = styled.h1`
+    position: absolute;
```

Comment:
In practice names like this are rarely given. It shoudl be more like StyledName or something but without element name

Reply:
fixed naming